### ISSUE TITLE
Лаг. Жуткий.

### ISSUE THREAD

**Issue by Hefren** [SBProX/SBProX-Dev#1401]
А что такое? До этого пользовался V2 лаг был, но вполне терпимый. Сейчас же просто ужас. И самое интересное что ресурсы компьютера то далеко не на полную грузятся. Так дальше и жить?

**Comment by Hefren** [comment]
Немного поэексперементировал. Упростил до невозможности. Самый большой лаг возникает на графиках типа "кластер дельта" и "объём дельта". Похоже у вас там чото дельту он как-то через одно место считает.

**Comment by joke22** [comment]
Согласен, лагает жёстко и котировка постоянно запаздывает на 1-30 секунд в моменте, потом догоняет... бред полный.

**Comment by SBProX** [comment]
В чем конкретно лаг?
Описывайте проблему, чтобы было понятно о чем идет речь.

**Comment by joke22** [comment]
> В чем конкретно лаг?
> Описывайте проблему, чтобы было понятно о чем идет речь.

Лично у меня:
1. Когда график перемещаешь по пространству, то он перемешается не плавно, а прерывисто (видимо, от истории зависит).
2. Когда цена ходит, то часто подтормаживает и потом догоняет (1-30 секунд). Начало торгов часто от 10сек до 1-2мин. Даже через коннектор КВИК на 1-2 секунды (но это уже, видимо, от пакетов зависит).
3. По мелочи... интерфейс сам по себе при нажатии кнопок заторможенно порой отвечает.

**Comment by SBProX** [comment]
> > В чем конкретно лаг?
> > Описывайте проблему, чтобы было понятно о чем идет речь.
> 
> Лично у меня:
> 
> 1. Когда график перемещаешь по пространству, то он перемешается не плавно, а прерывисто (видимо, от истории зависит).
> 2. Когда цена ходит, то часто подтормаживает и потом догоняет (1-30 секунд). Начало торгов часто от 10сек до 1-2мин. Даже через коннектор КВИК на 1-2 секунды (но это уже, видимо, от пакетов зависит).
> 3. По мелочи... интерфейс сам по себе при нажатии кнопок заторможенно порой отвечает.

1. Смотря что за графики и что на них отображено. Супер плавно нет ни в одной платформе. 
Можете выслать сюда скриншоты, чтобы было понятно.

2. Такая проблема есть, мы в курсе.

3. Смотря сколько у вас открыто окон, вкладок, и что на них стоит. Есть пользователи которые открывают десятки окон, 90 вкладок, с кучей индикаторов, и потом жалуются что у них все тормозит. Ну тут уж извините.
Надо понимать какая конфигурация у вас, может быть мы вам посоветуем что-то изменить в ней, и это поможет улучшить работу платформы.

**Comment by SBProX** [comment]
Касательно второго пункта: квик тоже не идеален, он и зависает, и тормозит на открытии в том числе.
